framework/include/material_image_helpers.hpp

```diff
@@ -375,6 +400,27 @@ namespace gvk
 		return result;
 	}
 
+	/**	This is a convenience method that allows to compile a selection of shared owning models and mesh indices.
+	 *	Valid usage means passing a model as parameter, and following it up with one or multiple mesh index parameters.
+	 *
+	 *	Model parameters must have shared ownership enabled.
+	 *	Mesh index parameters are supported in the forms:
+	 *	 - size_t
+	 *	 - std::vector<size_t>
+	 *
+	 *	Examples of valid parameters:
+	 *	 - make_models_and_meshes_selection(myModel, 1, 2, 3); // => a model and 3x size_t
+	 *	 - make_models_and_meshes_selection(myModel, 1, std::vector<size_t>{ 2, 3 }); // => a model and then 1x size_t, and 1x vector of size_t (containing two indices)
+	 *	 - make_models_and_meshes_selection(myModel, 1, myOtherModel, std::vector<size_t>{ 0, 1 }); // => a model and then 1x size_t; another model and 1x vector of size_t (containing two indices)
+	 */
+	template <typename... Args>
+	std::vector<std::tuple<avk::resource_ownership<gvk::model_t>, std::vector<mesh_index_t>>> make_shared_owning_models_and_meshes_selection(Args&... args)
```

Comment:
```suggestion
std::vector<std::tuple<avk::resource_ownership<gvk::model_t>, std::vector<mesh_index_t>>> make_selection_of_shared_models_and_mesh_indices(Args&... args)
```

I think, this would sound a bit nicer:
1. stating the action first
2. referring to "shared models" instead of "shared owning models", which is more concise and might also be less confusing
3. referring to "mesh indices" is more precise

If you agree, please rename and update the comments!

Reply:
yeh makes sense